### PR TITLE
New filters in Resources Status (22.10)

### PR DESCRIPTION
**Pull request by cg-tw**
## Description

New filters in Resources Status (22.10)

## Target version

- [ ] 20.10.x (staging)
- [ ] 21.04.x (staging)
- [ ] 21.10.x (staging)
- [ ] 22.04.x (staging)
- [ ] Cloud (staging)
- [ ] Plugin Packs (staging)
- [x] 22.10.x (next)

### CODE REVIEW

File: versioned_docs/version-22.10/alerts-notifications/resources-status.md [hunk 1]
```diff
@@ -120,6 +120,12 @@ You can filter the list of resources according to a number of predefined criteri
     - **Host group**
     - **Service group**
     - **Monitoring server**: resources monitored by a specific server (or poller)
+    - [**Host category**](../monitoring/categories.md)
+    - [Service category](../monitoring/categories.md)
```
Comment (mdebauchez) [inline versioned_docs/version-22.10/alerts-notifications/resources-status.md:124]: Filters name (line 123 to 128) should be in bold like the previous ones

File: versioned_docs/version-22.10/alerts-notifications/resources-status.md [hunk 2]
```diff
@@ -120,6 +120,12 @@ You can filter the list of resources according to a number of predefined criteri
...
+    - Host [severity](../monitoring/categories.md): name of the host severity
```
Comment (mdebauchez) [inline versioned_docs/version-22.10/alerts-notifications/resources-status.md:125]: Why the "Host" is not in the link like "Service Severity" line 127

File: i18n/fr/docusaurus-plugin-content-docs/version-22.10/alerts-notifications/resources-status.md [hunk 3]
```diff
@@ -124,6 +124,12 @@ Vous pouvez filtrer la liste des ressources selon un certain nombre de critères
     - **Groupes d'hôtes**
     - **Groupe de services**
     - **Serveur de supervision** : ressources supervisées par un serveur (ou collecteur) particulier.
+    - [**Catégorie d'hôte**](../monitoring/categories.md)
+    - [**Catégorie de service**](../monitoring/categories.md)
+    - [Criticité] d'hôte(../monitoring/categories.md): nom de la criticité de l'hôte
```
Comment (mdebauchez) [inline i18n/fr/docusaurus-plugin-content-docs/version-22.10/alerts-notifications/resources-status.md:129]: This link does not work and "d'hôte" should also be in the link

File: i18n/fr/docusaurus-plugin-content-docs/version-22.10/alerts-notifications/resources-status.md [hunk 4]
```diff
@@ -124,6 +124,12 @@ Vous pouvez filtrer la liste des ressources selon un certain nombre de critères
...
+    - Niveau de criticité d'hôte
+    - [Criticité du service](../monitoring/categories.md): nom de la criticité du service
+    - Niveau de criticité du service
```
Comment (mdebauchez) [inline i18n/fr/docusaurus-plugin-content-docs/version-22.10/alerts-notifications/resources-status.md:132]: Filters name (line 129 to 132) should be in bold like the previous ones

**Review by mdebauchez — CHANGES_REQUESTED** [review]

**Review by mdebauchez — APPROVED** [review]

**Review by tuntoja — APPROVED** [review]

**Pull request merged**